src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs

```diff
@@ -45,6 +49,12 @@ private static class EventIds
             EventIds.RequestEnd,
             "Received HTTP response headers after {ElapsedMilliseconds}ms - {StatusCode}");
 
+        private static readonly Action<ILogger, double, string?, Exception?> _requestFailed = LoggerMessage.Define<double, string?>(
+            LogLevel.Error,
+            EventIds.RequestFailed,
+            "Request failed after {ElapsedMilliseconds}ms, Error: {ErrorMessage}");
```

Comment:
1) Let's try to keep all the messages in the same style within the logging handler. The request is called "HTTP request" in other messages: `"HTTP request failed after {ElapsedMilliseconds}ms"`

2) I believe you don't need to additionally pass the ErrorMessage here. You are already passing the `Exception` object, so it will already be logged for you. E.g. you can check the [Logging in C# - Log exceptions](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#log-exceptions) section, and the example in the [High-performance logging - Define a logger message](https://learn.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging#define-a-logger-message) section.

Reply:
@CarnaViire done

Reply:
The message is not updated -- can you pls check (maybe the changes were not committed/pushed?)